crates/api_models/src/enums.rs

```diff
@@ -187,6 +191,7 @@ impl Connector {
                 | (Self::Payu, _)
                 | (Self::Trustpay, PaymentMethod::BankRedirect)
                 | (Self::Iatapay, _)
+                | (Self::Wellsfargopayout, _)
```

Comment:
This list is for payments, we can remove this from here

Reply:
removed